### ISSUE TITLE
[Broken Website] Beehiiv text unreadable

### ISSUE THREAD

**Issue by jonas-w** [darkreader/darkreader#11613]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [X] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

Here are a few random links:
https://fossweekly.beehiiv.com/p/foss-weekly-47-godot-4-1-new-linux-laptop-and-phone-gnome-45-china-s-distro-and-more
https://opensourcewatch.beehiiv.com/p/suse-go-private
https://jasonfeifer.beehiiv.com/p/how-to-solve-big-problems

Beehiiv seems to be a newsletter platform where everyone has their own subdomain.

Basically with darkreader enabled the background is a very dark grey and the text color is a bit lighter grey but not really readable.

### Website Address

*.beehiiv.com

### Steps To Reproduce

1. Go to the links provided
2. Can't read the text properly

### Screenshots

![image](https://github.com/darkreader/darkreader/assets/32615971/03f62cad-a3df-4bb6-82bc-eacd1eab469c)

### Operating System

Linux

### Web Browser name and version

Firefox 116

### Dark Reader version

4.9.65

**Comment by AeonEternum** [comment]
Can I bump this? This issue still persists in 2026 with Dark Reader version 4.9.123.

<img width="1080" height="1640" alt="Image" src="https://github.com/user-attachments/assets/2f637641-f214-4508-a0dd-090e68de2ce2" />
